pkg/kubelet/cm/node_container_manager_linux_test.go

```diff
@@ -400,3 +401,47 @@ func getEphemeralStorageResourceList(storage string) v1.ResourceList {
 	}
 	return res
 }
+
+func TestUpdateNodeAllocatableCgroups(t *testing.T) {
+	tests := []struct {
+		name             string
+		internalCapacity v1.ResourceList
+		updatedList      v1.ResourceList
+		expectedChange   bool
+	}{
+		{
+			name:             "Empty capacity",
+			internalCapacity: v1.ResourceList{},
+			updatedList:      v1.ResourceList{},
+		},
+		{
+			name:             "irrelevant capacity update",
+			internalCapacity: v1.ResourceList{},
+			updatedList:      getResourceList("10", "10Gi"),
+		},
+		{
+			name:             "CPU updated",
+			internalCapacity: getResourceList("12", "10Gi"),
+			updatedList:      getResourceList("10", "10Gi"),
+			expectedChange:   true,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			node := &v1.Node{}
+			node.Name = "test"
+			cm := &containerManagerImpl{
+				internalCapacity: test.internalCapacity.DeepCopy(),
+				nodeInfo:         node,
+			}
+
+			_ = cm.UpdateNodeAllocatableCgroups(test.updatedList)
+
+			if test.expectedChange != !apiequality.Semantic.DeepEqual(test.internalCapacity, cm.internalCapacity) {
```

Comment:
Nit: I think this could be slightly more readable if the result of `!apiequality.Semantic.DeepEqual` was assigned a descriptive variable name (i.e. `actualChange`).

Reply:
Thanks! Updated...